scripts/check-release.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const { version: releaseVersion } = require('../lerna.json')
+const { findBrowserSdkPackageJsonFiles, printLog, logAndExit, executeCommand } = require('./utils')
+
+async function main() {
+  // await checkGitTag()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏comment leftover after debug?

Reply:
yep 😄